### DISCUSSION TITLE
autostart behavior? (from #212)

### DISCUSSION THREAD

**Discussion by ousia** [Cimbali/pympress#261] [Q&A] [answered]
@Cimbali,

since #212 is limited to collaborators, I wanted to know how Pympress will handle media renditions with autostart.

I mean, I wonder whether it would honor only `/AA << /PC << /OP 1 >> /PO << /OP 0 >> >>` (play rendition when page is reached, stop it when page is left) or it would behave like Okular?

BTW, the `/A`  dictionary defines actions when the annotation is activated (https://opensource.adobe.com/dc-acrobat-sdk-docs/standards/pdfstandards/pdf/PDF32000_2008.pdf#page=417).

Many thanks for your help.

**Comment by Cimbali** [comment, marked as answer]
The goal is to behave according to spec, i.e. not like Okular. I’ve +1’d the [Okular issue](https://bugs.kde.org/show_bug.cgi?id=436709) so they hopefully fix it some day and made [a PR on popper to extend the glib interface](https://gitlab.freedesktop.org/poppler/poppler/-/merge_requests/1295) so we can have access to all the actions, additional actions, and other info needed to properly implement autostart. Once that’s merged I’ll write the correct autostart into pympress and users who want access to it will need to have a sufficiently recent libpoppler.

(PS. Sorry didn’t mean to lock #212)

**Reply by ousia** [reply]
Many thanks for your reply, @Cimbali.

All is clear to me now.
